asynceapi/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Initially written by Jeremy Schulman at https://github.com/jeremyschulman/aio-eapi
```

Comment:
Apache2 needs a double copyright one for Jeremy Schulman and ours in these files